lib/src/bazelify/build.dart

```diff
@@ -282,14 +285,21 @@ class DartWebApplication implements DartBuildRule {
         '    script_file = "$scriptFile",\n'
         '    deps = _PUB_DEPS';
     if (includeLibraries.isEmpty) {
-      return '$buffer,\n)';
+      buffer = '$buffer,\n)';
     } else {
-      return buffer +
-          ' + [\n' +
+      buffer += ' + [\n' +
           includeLibraries
               .map/*<String>*/((l) => '        ":${l.name}",\n')
               .join() +
           '    ],\n)';
     }
+    buffer += '\ndev_server(\n'
+        '    name = "${name}_serve",\n'
+        '    deps = _PUB_DEPS + [\n' +
```

Comment:
Can we leave out _PUB_DEPS if we are also depending on the library that already has _PUB_DEPS?